src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -125,10 +125,26 @@
     "drop": PREFIX + 239,
     "decision": PREFIX + 240,
     "keep": PREFIX + 241,
+    # Spans
+    "span.op": PREFIX + 242,
+    "span.description": PREFIX + 243,
+    "span.module": PREFIX + 244,
+    "span.domain": PREFIX + 245,
     # GENERAL/MISC (don't have a category)
     "": PREFIX + 1000,
 }
-SHARED_STRINGS = {**SESSION_METRIC_NAMES, **TRANSACTION_METRICS_NAMES, **SHARED_TAG_STRINGS}
+
+# 300-399
```

Comment:
Since we're already at +245 in tag keys, I can start span metrics at +400 to give some more slack.